installer/packer/scripts/systemd/vic-service.target

```diff
@@ -0,0 +1,8 @@
+[Unit]
```

Comment:
to group services together thus we have a synchronization point to wait on if we want to do tests on interactions between services. If this is unnecessary, I'll remove it.